src/core.js

```diff
@@ -155,6 +155,9 @@ define([
                 }
                 _this.ensureCurrentMugIsSaved(function () {
                     _this.validateAndSaveXForm(forceFullSave);
+                    if (typeof window.analytics !== "undefined") {
+                        window.analytics.track("Clicked Save in Formbuilder");
+                    }
```

Comment:
I would move this above the call to `_this.validateAndSaveXForm` in case it raised an exception.